### PR TITLE
Node week 3 refinements

### PR DESCRIPTION
**Pull request by sansaid**
## What does this change?

Module: Node
Week(s): 3

## Description

* Fixing some grammatical errors
* Updating Heroku guide to include how to install on WSL and changed `master` to `main`
* Added an exercise for students to work on creating a PUT route with the `albums` exercise
* Fixed instructions for the `quotes` exercise so that it is clear it will be used as a teacher led exercise

## Who needs to know about this?

@ChrisOwen101 @SallyMcGrath

## Rendered Pages

-----
[View rendered docs/node/week-3/lesson.md](https://github.com/sansaid/syllabus/blob/node-week-3-refinements/docs/node/week-3/lesson.md)

### CODE REVIEW

File: docs/node/week-3/lesson.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ So what will we build? we will build a **CRUD** API. CRUD stands for Create, Ret
 
 ### 2.1) Using Update
 
-This is an Teacher led exercise which can be used to show how we might retrieve an element by ID using a GET request.
+This is a teacher led exercise which can be used to remind students how update resources by making a PUT request.
```
Comment (textbook) [inline docs/node/week-3/lesson.md]:
```suggestion
This is a teacher led exercise which can be used to remind students how to update resources by making a PUT request.
```

File: docs/node/week-3/lesson.md [hunk 2]
```diff
@@ -54,13 +54,13 @@ When you remix the starting project, immediately rename it as your own.
 
 #### 2.2 Workshop: Updating Data
 
-Lets look back at our original objectives.
+Let's look back at our original objectives using the [cyf-albums-start](https://glitch.com/~cyf-album-put-start) project. Try to apply what you learned about PUT routes to this project. Remember to remix and rename the project as your own.
```
Comment (textbook) [inline docs/node/week-3/lesson.md]: I don't know if I need to be in some kind of Glitch team/org to see it, but that URL gives me:

> We didn't find cyf-album-put-start

**Review by textbook — APPROVED** [review]
Looks good generally, just need to double-check the Glitch URL otherwise 👍

File: docs/node/week-3/lesson.md (hunk 2 above)
Comment (sansaid) [reply docs/node/week-3/lesson.md]: Hmm, strange - thanks for checking that! Let me see if I've set something up incorrectly

Comment (sansaid) [reply docs/node/week-3/lesson.md]: I updated the URL and didn't update the docs 🙃 Thanks for spotting @textbook !

**Pull request merged**